### PR TITLE
Added Package for admin panel

### PR DESCRIPTION
**Pull request by sheeshmohsin**
Currently edit and delete option is working for two tables only, will push the code of edit and delete option for other four tables tomorrow.

### CODE REVIEW

File: app/both/_config/adminConfig.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
+AdminConfig = {
+  name: 'Waartaa',
+  adminEmails: ['sheeshmohsin@gmail.com'],
```
Comment (rtnpro) [inline app/both/_config/adminConfig.js]: We should not hard code such values. We should pick it up from `Meteor.settings`

**Comment by rtnpro** [comment]
@sheeshmohsin Can we have this code inside a package, say: `waartaa/app/packages/admin`?

**Comment by sheeshmohsin** [comment]
If it's possible to have this code inside a package, then please guide me in doing this.

**Comment by sheeshmohsin** [comment]
Please check this pull request.

**Pull request closed without merging**